backends/pirateweather.net.go

```diff
@@ -0,0 +1,152 @@
+package backends
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/schachmat/wego/iface"
+)
+
+const (
+	pirateweatherURI = "https://api.pirateweather.net/forecast/"
+)
+
+type PirateweatherConfig struct {
+	apiKey string
+	debug  bool
+}
+
+func (c *PirateweatherConfig) Setup() {
+	flag.StringVar(&c.apiKey, "pirateweather-api-key", "", "pirateweather backend: the api `KEY` to use")
+	flag.BoolVar(&c.debug, "pirateweather-debug", true, "pirateweather backend: print raw request and responses")
+}
+
+func (c *PirateweatherConfig) Fetch(location string, numdays int) iface.Data {
+	if c.debug {
+		log.Printf("pirateweather location %v", location)
+	}
+
+	res := iface.Data{}
+	reqURI := fmt.Sprintf("%s/%s/%s", pirateweatherURI, c.apiKey, location)
```

Comment:
The URL already has a trailing slash. So either remove it in the URL or here after the first %s.